app/src/main/java/org/dhis2/utils/granularsync/SyncStatusDialogNavigator.kt

```diff
@@ -50,6 +50,10 @@ class SyncStatusDialogNavigator(
                 navigateToEnrollmentFormScreen(syncStatusItem.type as SyncStatusType.Enrollment)
         }
 
+        if (intent?.hasExtra(OPEN_ERROR_LOCATION) == false) {
```

Comment:
Does this dismiss the dialog when trying to open without errors?

Reply:
No, this is for dismiss unless it navigates to the last step, the issue location